cajita/src/Cajita_GlobalMesh.cpp

```diff
@@ -14,13 +14,18 @@
 namespace Cajita
 {
 
-template class GlobalMesh<UniformMesh<float>>;
-template class GlobalMesh<UniformMesh<double>>;
+#define CAJITA_INST_GLOBALMESH( MESH, FP, NSD )                                \
```

Comment:
Your description mentioned removing all explicit instantiation. Did you mean to keep this? Or keep `GlobalGrid`? `LocalGrid` seems to be the only one too large to manage

Reply:
Ah missed this - just deleted that file.